Topics/System_Design/DesignPatterns/Creational_Design_Patterns.md

```diff
@@ -0,0 +1,126 @@
+# Creational Design Patterns
```

Comment:
I would try to add a table of content which would be good to jump from section to section.